catalyst/dl/callbacks/metrics/cmc.py

```diff
@@ -18,78 +18,6 @@ class CMCScoreCallback(Callback):
     You should use it with `ControlFlowCallback`
     and add all query/gallery sets to loaders.
     Loaders should contain "is_query" and "label" key.
-
-    .. code-block:: python
```

Comment:
could we add some minimal example here?

Reply:
That example is outdated, that is why it was deleted.

And I think that example for this Callback is too large for docs. I suggest to paste link to MetricLearningMinimalExample into the doc. I suppose It will be easier to keep actual example in the one place.

Reply:
added link to example